index.rst

```diff
@@ -14,36 +14,53 @@ Quick Start
 Here are the basic steps needed to get :ref:`set up <setup>` and contribute a
 patch:
 
-1. :ref:`Get the source code <checkout>`::
+1. Set up and install dependencies.
+
+   Install :ref:`Mercurial <vcsetup>` and other dependencies.
+
+   The dependencies needed will depend on the platform you're on.
+   Go to :ref:`Get Setup <setup>` page for detailed information.
+
+2. :ref:`Get the source code <checkout>`::
 
       hg clone https://hg.python.org/cpython
 
-   See :ref:`version control setup <vcsetup>` for installing Mercurial.
 
-2. :ref:`Build Python <compiling>`.  On all platforms, install build
-   dependencies (such as compilers). On :ref:`UNIX <unix-compiling>`
-   (including Mac OS X)::
+3. Build Python.
+
+   Detailed information can be found :ref:`here <compiling>`.
+   There are different instructions for :ref:`UNIX <unix-compiling>`,
+   :ref:`Mac OS <MacOS>`, and :ref:`Windows <windows-compiling>`.
+
+   The command to compile on UNIX is::
+
+      ./configure
+      make -j2
+
+   On some Mac OS:::
 
-      ./configure --with-pydebug && make -j2
+      CPPFLAGS="-I$(brew --prefix openssl)/include" \
```

Comment:
@Mariatta Thanks for the updates! Perhaps it would be better to mention that for OS X that the developer should refer to the section in `setup.rst` to set CPPFLAGS and LDFLAGS. This way, `setup.rst` can have details whether using brew or macports, but the user is notified that an additional step may be needed. As written and pointed out be @berkerpeksag, a user with macports would get an error if setting to these brew parameters. What do you think?